docs/cloud/features/scheduler/dagster.md

```diff
@@ -1,3 +1,334 @@
+
+## Setup
+
+Your SQLMesh project must be configured and connected to Tobiko Cloud before using the Dagster integration.
+
+Learn more about connecting to Tobiko Cloud in the [Getting Started page](../../tcloud_getting_started.md).
+
+### Install libraries
+
+After connecting your project to Tobiko Cloud, you're ready to set up the Dagster integration.
+
+Start by adding the `tobiko-cloud-scheduler-facade` library to your [Dagster project](https://docs.dagster.io/guides/understanding-dagster-project-files).
+
+To do this, add it to the `install_requires` section of `setup.py`:
+
+```python title="setup.py" hl_lines="3"
+install_requires=[
+    "dagster",
+    "tobiko-cloud-scheduler-facade[dagster]"
+],
+```
+
+### Connect Dagster to Tobiko Cloud
+
+Dagster recommends [injecting secret values using Environment Variables](https://docs.dagster.io/guides/dagster/using-environment-variables-and-secrets#using-environment-variables-and-secrets). The exact method is specific to your Dagster deployment.
+
+Here, we will describe the method for **local development**.
+
+In your Dagster project, add the following entries to the `.env` file (creating it if it doesnt exist):
+
+```sh title=".env"
+TOBIKO_CLOUD_BASE_URL=<URL for your Tobiko Cloud project>
+TOBIKO_CLOUD_TOKEN=<your Tobiko Cloud API token>
+```
+
+The base URL and password values will be provided to you during your Tobiko Cloud onboarding.
+
+### Create Dagster objects
+
+You are now ready to create Dagster objects connected to Tobiko Cloud.
+
+This example code demonstrates the creation process, which requires:
+
+- Importing the `SQLMeshEnterpriseDagster` class
+- Creating a `SQLMeshEnterpriseDagster` instance configured with the environment variables you declared above
+- Creating the `Definitions` object with the `create_definitions()` method
+
+In the `definitions.py` file in your project, insert the following:
+
+```python title="definitions.py" linenums="1"
+from dagster import EnvVar
+from tobikodata.scheduler_facades.dagster import SQLMeshEnterpriseDagster
+
+sqlmesh = SQLMeshEnterpriseDagster(
+    url=EnvVar("TOBIKO_CLOUD_BASE_URL").get_value(),
+    token=EnvVar("TOBIKO_CLOUD_TOKEN").get_value(),
+)
+
+tobiko_cloud_definitions = sqlmesh.create_definitions(environment="prod")
+```
+
+!!! info
+    If there is an existing definitions object already declared in your Dagster project, you can merge in the Tobiko Cloud definitions like so:
+
+    ```python
+    defs = Definitions(...)
+
+    defs.merge(tobiko_cloud_definitions)
+    ```
+
+This is all that's needed to integrate with Tobiko Cloud!
+
+Once Dagster loads your project, you will see some new objects become available.
+
+## Available Dagster objects
+
+The Dagster integration exports the following:
+
+- An `Asset` object for every SQLMesh Model
+  ![Dagster UI Asset Lineage](./dagster/asset_lineage.png)
```

Comment:
copy and paste this image up top after the intro sentence. Get the reader excited about the outcome they want to achieve.